fastchat/serve/gradio_block_arena_anony.py

```diff
@@ -33,24 +33,27 @@
     acknowledgment_md,
     get_ip,
     get_model_description_md,
+    _write_to_json,
 )
+from fastchat.serve.moderation.moderator import AzureAndOpenAIContentModerator
 from fastchat.serve.remote_logger import get_remote_logger
 from fastchat.utils import (
     build_logger,
-    moderation_filter,
 )
 
 logger = build_logger("gradio_web_server_multi", "gradio_web_server_multi.log")
 
 num_sides = 2
 enable_moderation = False
+use_remote_storage = False
```

Comment:
why do we need this be global variable?

Reply:
I think globally False is the correct decision because we have a set bucket where we place images and not everyone will do it that way - i think that having it default False makes it so anyone can run this without google cloud storage.